src/browserview/BrowserComponent.java

```diff
@@ -252,9 +252,11 @@ public void jumpToComment(){
 	
 	private boolean isBrowserActive(){
 		if (driver.getCurrentUrl().isEmpty() && driver != null){
+			logger.warn("Unable to read url from bview. Resetting.");
 			return false;
 		}
 		else if (driver == null){
+			logger.warn("chromedriver process was killed !");
 			return false;
 		}
```

Comment:
The logic here could be structured like this:

```java
if (driver == null) {
    // chromedriver process killed
}
// driver != null from here on
else if (driver.getCurrentUrl() == null /* extra null check */ || driver.getCurrentUrl().isEmpty()) {
    // unable to read url from browser
}
```

Also, is it possible that `getCurrentUrl()` may throw `WebDriverException`? I looked at the docs and there was no mention of it, but then `WebDriverException` is unchecked, so...